.gitignore

```diff
@@ -0,0 +1,6 @@
+query.log
```

Comment:
There was none in the parent pom repo.  Grabbed the one from metrics and removed the non-maven stuff from it.